Source/items.cpp

```diff
@@ -3226,7 +3228,7 @@ Item *SpawnUnique(_unique_items uid, Point position, bool sendmsg /*= true*/)
 	return &item;
 }
 
-void SpawnItem(Monster &monster, Point position, bool sendmsg)
+void SpawnItem(Monster &monster, Point position, bool sendmsg, bool spawn /*= false*/)

```

Comment:
SpawnItem() having an option not to spawn confused me, but it's only related to spawning on other clients now that i read it while awake.

Reply:
It's still confusing that's why I mentioned a possible refactoring in the notes 🙂